.github/workflows/push.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: |
```

Comment:
CI should have a step before this to clone and install KlepsydraSDK

Reply:
Not modified in this PR, @tomasz3dk?

Reply:
If you don't add it, the CI will always fail :)

Reply:
Done, very hand-made, please review.